docs/articles/documentation/test-api/switching-between-iframes.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: docs
+title: Switching Between iframes
+permalink: /documentation/test-api/switching-between-iframes.html
+---
+# Switching Between \<iframes\>
+
+At any moment, a TestCafe test has its scope limited to a single \<iframe\>. If a webpage contains
+several \<iframes\> and you need to use more than one in a test, you will need to switch between them.
```

Comment:
A single iframe on the page is enough to use this (discussed in private with @VasilyStrelyaev )